std/datetime/date.d

```diff
@@ -3055,20 +3055,19 @@ public:
         if (isSomeString!S)
     {
         import std.algorithm.searching : countUntil;
-        import std.conv : to;
         import std.exception : enforce;
         import std.format : format;
         import std.string : strip;
 
-        immutable dstr = to!dstring(strip(isoString));
+        immutable str = strip(isoString);
```

Comment:
I was thinking that `strip` and  `countUntil` both did, but if they don't, then that's not an issue (though actually, I should probably know for `strip`, since I did a rewrite of that at one point - but I guess that it was long enough ago that I forgot).